api/src/infrastructure/infrastructure.rs

```diff
@@ -65,13 +65,13 @@ pub trait Infrastructure: Send + Sync {
     async fn stop_services(
         &self,
         status_id: &String,
```

Comment:
We can address the clippy suggestions for these 3 functions by changing &String to &str.

Reply:
Done.